test/IRGen/unmanaged_objc_throw_func.swift

```diff
@@ -0,0 +1,70 @@
+// RUN: %target-swift-frontend -emit-ir %s | %FileCheck %s
```

Comment:
Should mark this test as `REQUIRES: objc_interop`, since it's using the real Foundation and CF.